### PR TITLE
tproxyUpstreamConnMark

### PR DESCRIPTION
**Pull request by tomastigera**
## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: pkg/apis/projectcalico/v3/felixconfig.go
Comment (fasaxc) [inline pkg/apis/projectcalico/v3/felixconfig.go:701]: Typo: `s/tell/tells/`

**Review by fasaxc — APPROVED** [review]
One typo but LGTM

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Comment by marvin-tigera** [comment]
Removing "merge-when-ready" label due to new commits

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
